extensions/wikia/Scribe/ScribeEventProducer.class.php

```diff
@@ -5,6 +5,7 @@
 class ScribeEventProducer {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Class "ScribeEventProducer" has 38 methods, which is greater than 20 authorized. Split it into smaller classes. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1448)